beacon-chain/rpc/proposer_server.go

```diff
@@ -332,6 +322,30 @@ func (ps *ProposerServer) deposits(ctx context.Context, currentVote *ethpb.Eth1D
 	return pendingDeposits, nil
 }
 
+// determineLatestETH1Height determines what the latest eth1Blockhash is by tallying the votes in the
+// beacon state
+func (ps *ProposerServer) determineLatestETH1Height(ctx context.Context, beaconState *pbp2p.BeaconState,
```

Comment:
suggest changing the name to `latestEth1Height`, the word `determine` sounds a bit redundant there

```suggestion
func (ps *ProposerServer) latestEth1Height(ctx context.Context, beaconState *pbp2p.BeaconState,
```